py/desispec/fluxcalibration.py

```diff
@@ -102,36 +84,100 @@ def convolveModel(wave,resolution,flux):
 
         return convolved
 
+    Chisq=1e100
+    bchisq=0
+    rchisq=0
+    zchisq=0
     nstd=stdflux.shape[0]
     nstdwave=stdwave.shape[0]
     maxDelta=1e100
     bestId=-1
     red_Chisq=-1.
 
-    for i in range(nstd):
+    # search stellar template metadata for canonical f-type model
+
+    tcm=np.where(teff==6000.0)[0]
+    lcm=np.where(logg==4.0)[0]
+    fcm=np.where(feh==-1.5)[0]
+    can_mod=list(set(tcm).intersection(set(lcm).intersection(fcm)))[0]
```

Comment:
This appears to be picking only one point in the standard star phase space rather than considering multiple standard star options and finding the one that fits best.